core/lib/dal/src/contract_verification_dal.rs

```diff
@@ -40,59 +39,55 @@ impl Display for Compiler {
 }
 
 impl ContractVerificationDal<'_, '_> {
-    pub async fn get_count_of_queued_verification_requests(&mut self) -> Result<usize, SqlxError> {
-        {
-            sqlx::query!(
-                r#"
-                SELECT COUNT(*) as "count!"
-                FROM contract_verification_requests
-                WHERE status = 'queued'
-                "#
-            )
-            .fetch_one(self.storage.conn())
-            .await
-            .map(|row| row.count as usize)
-        }
+    pub async fn get_count_of_queued_verification_requests(&mut self) -> sqlx::Result<usize> {
+        sqlx::query!(
+            r#"
+            SELECT COUNT(*) as "count!"
+            FROM contract_verification_requests
+            WHERE status = 'queued'
+            "#
+        )
```

Comment:
Nit: Since you're changing SQL query formatting from the `sqlx` perspective, it may make sense to apply "format-proofing" as in most newer queries (e.g., [here](https://github.com/matter-labs/zksync-era/blob/756ba6cbc5b0ecbcb336a6082ea61ab606f5fd82/core/lib/dal/src/blocks_dal.rs#L153)). This would make queries resistant to changes in the surrounding formatting and would make it easier to search queries in `sqlx-data.json`.

Reply:
done